arrocke/gloss-translation
# docs: Improve local dev instructions

## What has changed


Placed nx global instruction line before npm install line.

## Connected Issues


closes #287 
## QA Steps


Review files changes and ensure that: 
*In a terminal run: npm i -g nx to make the nx command available in a terminal.*
is before: 
*In a terminal run: npm install to install all of the project's dependencies.*

## Post-Deployment



- [ x] Nothing required
